python/Makefile.am

```diff
@@ -0,0 +1,34 @@
+SWIG_SRC = meep.i numpy.i vec.i
+EXTRA_DIST = $(SWIG_SRC)
+HPPFILES=	                          \
+ $(top_srcdir)/src/meep.hpp               \
+ $(top_srcdir)/src/meep_internals.hpp     \
+ $(top_srcdir)/src/bicgstab.hpp           \
+ $(top_srcdir)/src/meep/vec.hpp           \
+ $(top_srcdir)/src/meep/mympi.hpp
+
+BUILT_SOURCES = meep-python.cpp meep.py
+
+AM_CPPFLAGS = -I$(top_srcdir)/src                   \
+              -I$(top_srcdir)/src/meep              \
```

Comment:
`-Isrc/meep` shouldn't be necessary since we always `#include <meep/vec.hpp>` etcetera.